ontology/yaml/resources/HVAC/entity_types/CH.yaml

```diff
@@ -607,3 +607,9 @@ CH_SS_CHWSWISOVM_CWSISOVM_CWRISOVM_CHWRWISOVM_CHWDT:
   opt_uses:
   - lost_power_alarm
   - run_mode
+  
+CH_CWDT:
```

Comment:
> s nothing else on this chiller except condenser water monitoring? that seems incomplete. what fields should it have?

Here, It monitors, Condensing (supply, Return) water monitoring. Chilled supply and return water monitoring.
Since, CHWDT had  linked to chilled_supply_water_temperature_sensor already had CH_SS_CHWDT as canonical. another canonical specifically for the condensing water was created. However, now, it has been modified to a new canonical CH_CWDT_CHWDT having both chilled and condensing water temperature monitoring.